ftdc/cmd/parser.go

```diff
@@ -165,25 +194,19 @@ func (gpw *gnuplotWriter) CompileAndClose() string {
 
 func main() {
 	if len(os.Args) < 2 {
-		// We are a CLI, it's appropriate to write to stdout.
-		//
-
 		nolintPrintln("Expected an FTDC filename. E.g: go run parser.go <path-to>/viam-server.ftdc")
 		return
 	}
 
 	ftdcFile, err := os.Open(os.Args[1])
 	if err != nil {
-		// We are a CLI, it's appropriate to write to stdout.
-		//
-
 		nolintPrintln("Error opening file. File:", os.Args[1], "Err:", err)
-
 		nolintPrintln("Expected an FTDC filename. E.g: go run parser.go <path-to>/viam-server.ftdc")
 		return
 	}
 
-	data, err := ftdc.Parse(ftdcFile)
+	logger := logging.NewLogger("parser")
+	data, err := ftdc.ParseWithLogger(ftdcFile, logger)
```

Comment:
Does the parser `main` program here get a lot of info-level-and-above logs now?

Reply:
IIRC, there are no info logs. Just debug for item by item details and warn/error for problems.